src/main/java/org/folio/circulation/support/JsonPropertyFetcher.java

```diff
@@ -194,6 +195,13 @@ public static void copyProperty(
     if (from.containsKey(propertyName)) {
       to.put(propertyName, from.getValue(propertyName));
     }
+  }
+
+  public static Period getPeriodProperty(JsonObject json, String propertyName, Period def) {
+    if (json == null || !json.containsKey(propertyName)) {
+      return def;
+    }
 
+    return Period.from(json.getJsonObject(propertyName));
   }
```

Comment:
I don't think it is a good idea to introduce a dependency on domain classes in this class. 

I think it would be more appropriate to either provide an overload of the `from` method or (preferably) introduce a dedicated JSON to period mapper for this.

Reply:
This method has been moved to the LostItemPolicy class.